clang/test/CodeGenSYCL/infer-address-spaces.cpp

```diff
@@ -17,5 +17,5 @@ void foo(const float *usm_in, float* usm_out) {
 
 // No addrspacecast before loading and storing values
 // CHECK-NOT: addrspacecast
-// CHECK:  %0 = load float, ptr addrspace(1)
-// CHECK:  store float %0, ptr addrspace(1)
+// CHECK:  %__SYCLKernel.ascast.val1.val = load float, ptr addrspace(1)
```

Comment:
To avoid the same problem in the future, how about using a regexp here?

Reply:
Agree, done in ed6d2e0e542751da0aba05421d9719ffe724c9ec